src/components/PlaylistPage/CourseInfo.js

```diff
@@ -1,21 +1,19 @@
 import React, {PropTypes} from 'react';
 import {connect} from 'react-redux';
-import {getCourseInfo} from '../../util';
+import {getCourseInfoMarkup} from '../../util';
 import {getTranslator} from '../../selectors/translate';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
 import styles from './CourseInfo.scss';
 
 const CourseInfo = React.createClass({
   render() {
-    const {t} = this.props;
-    const courseName = this.props.courseName;
-    const courseInfo = getCourseInfo(courseName);
-    return (      
-      <div className={this.props.isStudentMode ? styles.containerStudent : styles.containerTeacher}>
+    const {t, isStudentMode, courseInfo} = this.props;
+    return (
+      <div className={isStudentMode ? styles.containerStudent : styles.containerTeacher}>
         {courseInfo ?
```

Comment:
You are checking if `courseInfo` exists here, but will that happen? It must be if there is no index.md, but if that is the case, won't getCourseInfoMarkup throw an exception in the catch-clause, and we never get here?